scripts/getDivviIntegrators.ts

```diff
@@ -7,12 +7,20 @@ import { getHyperSyncClient } from './utils'
 import { NetworkId } from './types'
 import { divviRegistryAbi } from '../abis/DivviRegistry'
 import { rewardPoolAbi } from '../abis/RewardPool'
+import { fetchWithTimeout } from './utils/fetchWithTimeout'
 
 const DIVVI_REGISTRY_CONTRACT_ADDRESS =
   '0xEdb51A8C390fC84B1c2a40e0AE9C9882Fa7b7277'
 const DIVVI_INTEGRATION_REWARDS_ENTITY =
   '0x6226ddE08402642964f9A6de844ea3116F0dFc7e'
 
+const WHITELIST_URL =
```

Comment:
nit: consider replacing `whitelist` with `allowlist` to align with inclusive language best practices. It’s clearer and more widely adopted across the industry now.